apis/r/tests/testthat/test-SOMADataFrame.R

```diff
@@ -32,9 +30,13 @@ test_that("SOMADataFrame creation", {
     "All columns in 'values' must be defined in the schema"
   )
 
-  tbl0 <- arrow::arrow_table(
-    foo = 1L:10L, bar = 1.1:10.1, baz = letters[1:10]
-  )
+  tbl0 <- arrow::arrow_table(foo = 1L:10L,
+                             bar = 1.1:10.1,
+                             baz = letters[1:10],
+                             schema=arrow::schema(arrow::field("foo", arrow::int32(), nullable=FALSE),
+                                                  arrow::field("bar", arrow::float64(), nullable=FALSE),
+                                                  arrow::field("baz", arrow::large_utf8(), nullable=FALSE)))
+
```

Comment:
Any reason not to simplify and re-use the same schema defined above?

Reply:
Yes, I showed you the error over slack.  `large_utf8` is needed.

Reply:
Yup, and your follow-up helps.